cfy_manager/main.py

```diff
@@ -87,15 +89,16 @@
     copy,
     can_lookup_hostname,
     is_installed,
-    is_dir_empty,
+    is_all_in_one_manager,
     get_installed_services_names,
 )
 from .utils.install import is_premium_installed, yum_install, yum_remove
 from .utils.files import (
     remove as _remove,
     remove_temp_files,
     touch,
-    read_yaml_file
+    read_yaml_file,
+    update_yaml_file
```

Comment:
Nitpick: Good to have a , on the last line of a list- if you'd done so previously then adding the new entry would've been a one line change instead of a fake two line change.

Reply:
Thanks! Will keep that in mind.